### PR TITLE
Cristina Savrin Fourth Automation Task (POM)

### PR DESCRIPTION
**Pull request by CristinaSavrin**

### CODE REVIEW

File: src/test/java/cristina_savrin/UIStepDefinition.java [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    @Then("Verify default text disappears")
+    public void defaultTextDisappears() {
+        log.debug("Verify that default text" + addQuotes("No rows. Try reset filters") + " disappears");
+        assertThat(page.noPetsText.size(), equalTo(0));
+    }
+
+    @Given("Add random pets")
+    public void addRandomPets() {
+
+        int petsNum = 5;
```
Comment (dgrimaliuc) [inline src/test/java/cristina_savrin/UIStepDefinition.java:119]: num shold be provided as parameter in this step

File: src/test/java/cristina_savrin/UIStepDefinition.java [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        driver.close();
+    }
+
+    @Before
+    public void before() {
+
+        var pathToChrome = "src/main/resources/chromedriver.exe";
+        System.setProperty("webdriver.chrome.driver", pathToChrome);
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--remote-debugging-port=9222");
```
Comment (dgrimaliuc) [inline src/test/java/cristina_savrin/UIStepDefinition.java:42]: Can you explin why is used for remote-debugging-port ?

File: src/test/java/cristina_savrin/UIStepDefinition.java [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+    @And("Check [Adopt Selected Pets] and [Deselect] buttons are disabled by default")
+    public void checkAdoptSelectedPetsAndDeselectButtonsAreDisabledByDefault() {
+
+        log.debug("Verify [Adopt Selected Pets] and [Deselect] buttons are disabled by default");
+        assertThat(page.adoptBtn.isEnabled(), equalTo(false));
+        assertThat(page.deselectBtn.isEnabled(), equalTo(false));
+    }
+
+    @And("Adopt a pet")
+    public void adoptAPet() {
```
Comment (dgrimaliuc) [inline src/test/java/cristina_savrin/UIStepDefinition.java:134]: I think you shoud add num of pets as parameter

File: src/test/java/cristina_savrin/UIStepDefinition.java (hunk 2 above)
Comment (CristinaSavrin) [reply src/test/java/cristina_savrin/UIStepDefinition.java:42]: I had an issue with ChromeDriver, tried several workarounds from internet and this was the one that helped. 

image

Maybe there is another way..

**Review by dgrimaliuc-crunchyroll — APPROVED** [review]

**Pull request merged**